deps.bzl

```diff
@@ -2659,15 +2659,27 @@ def install_buildbuddy_dependencies(workspace_name = "buildbuddy"):
         version = "v3.0.0-20200313102051-9f266ea9e77c",
     )
     http_file(
-        name = "io_bazel_bazel-3.7.0-darwin-x86_64",
-        sha256 = "54657467a38db95c6063692315fcdb59817688254eff39333935d5e356675ebd",
-        urls = ["https://github.com/bazelbuild/bazel/releases/download/3.7.0/bazel-3.7.0-darwin-x86_64"],
+        name = "io_bazel_bazel-3.7-darwin-x86_64",
+        sha256 = "80c82e93a12ba30021692b11c78007807e82383a673be1602573b944beb359ab",
+        urls = ["https://github.com/bazelbuild/bazel/releases/download/3.7.2/bazel-3.7.2-darwin-x86_64"],
         executable = True,
     )
     http_file(
-        name = "io_bazel_bazel-3.7.0-linux-x86_64",
-        sha256 = "b7583eec83cc38302997098a40b8c870c37e0ab971a83cb3364c754a178b74ec",
-        urls = ["https://github.com/bazelbuild/bazel/releases/download/3.7.0/bazel-3.7.0-linux-x86_64"],
+        name = "io_bazel_bazel-3.7-linux-x86_64",
+        sha256 = "70dc0bee198a4c3d332925a32d464d9036a831977501f66d4996854ad4e4fc0d",
+        urls = ["https://github.com/bazelbuild/bazel/releases/download/3.7.2/bazel-3.7.2-linux-x86_64"],
+        executable = True,
+    )
```

Comment:
Do the tests pass if globally replacing 3.7 with 4.1? (I don't think there's a specific reason I chose 3.7 originally, other than that it matched our .bazelversion, so I'd be fine to use 4.1 everywhere)

Reply:
I want to test with whatever versions customers are using out in the wild (which could be 3.7 & 4.1), but I don't have any good sense of what customers are actually using today. Maybe @tylerwilliams  knows.
Unfortunately it doesn't look like this information is easy to query right now from our DB.

Reply:
Done, https://github.com/buildbuddy-io/buildbuddy-internal/issues/630